packit/config/job_config.py

```diff
@@ -35,19 +35,14 @@
 
 
 class JobType(Enum):
-    """ Type of the job to execute: pick the correct handler """
+    """ Type of the job to execute set by user in the config """
 
     propose_downstream = "propose_downstream"
     build = "build"
     sync_from_downstream = "sync_from_downstream"
     copr_build = "copr_build"
     production_build = "production_build"  # koji build
-    add_to_whitelist = "add_to_whitelist"
     tests = "tests"
-    report_test_results = "report_test_results"
-    pull_request_action = "pull_request_action"
-    copr_build_finished = "copr_build_finished"
-    copr_build_started = "copr_build_started"
     create_bugzilla = "create_bugzilla"
```

Comment:
Not sure about this one, it could be removed easily, is it used in the config - https://github.com/packit/packit-service/blob/717631ffd3d15ca76f0f050c803a7353f04c5f3c/packit_service/worker/jobs.py#L417?

Reply:
I found a usage here: https://github.com/packit/packit-service/blob/717631ffd3d15ca76f0f050c803a7353f04c5f3c/packit_service/worker/handlers/pagure_handlers.py#L47

Reply:
Yes, but removing that line shouldn't change anything, since we assign handler for the `PullRequestLabelPagureEvent` directly (in the link I provided) and not really use the mapping in that case, if I understand it correctly.

Reply:
Yes, it's not used in the mapping. The question is, do we potentially want this to be configurable anyhow?

If no, let's remove. If yes, let's keep it there.